manifests/repo.pp

```diff
@@ -38,7 +38,7 @@
   Optional[String[1]] $proxy_password   = undef,
 ) {
   if $version == undef and $repo_location == undef {
-    fail('`version` or `repo_location` is required')
+    $version = '5.0'
   }
   if $version != undef and $repo_location != undef {
```

Comment:
@mikhaildudzin please have a look at #779. I have removed this test so that $version is just ignored when $repo_location is set.